app/controllers/admin/lead_providers_controller.rb

```diff
@@ -1,16 +1,21 @@
 # frozen_string_literal: true
 
 class Admin::LeadProvidersController < Admin::BaseController
+  skip_after_action :verify_authorized, only: :index
+  skip_after_action :verify_policy_scoped, except: :index
+
   def index
-    @lead_providers = LeadProvider.all
+    @lead_providers = policy_scope(LeadProvider)
```

Comment:
`policy_scope` I imagine loads all lead providers that the current user can see?

Reply:
Correct